tutorials/svelte-pagination/src/App.svelte

```diff
@@ -0,0 +1,65 @@
+<script>
+	import {Allbooks} from './data'
+	import Book from './components/Book.svelte';
+</script>
+
+<main>
+	<div class='books'>
+		{#each Allbooks as book}
```

Comment:
This formatting is a little strange. Did you mean to convert to tabs here? Also it's not consistent with other files.

Reply:
Sorry, let me fix that.

Reply:
For some reason it looks strange on GitHub. Buy on my local machine its fine. I wonder why

Reply:
Someone else had this issue recently, I think it's due to some setting on VS Code. Please invest some time in fixing it Tammi.